functions-methods.js

```diff
@@ -33,4 +47,34 @@
 // checkEmailValidity("tessmellink@novi.nl") geeft true - want @ en punt op de juiste plek
 // checkEmailValidity("n.eekenanovi.nl") geeft false - want geen @
 // checkEmailValidity("n.eeken@novinl.") geeft false - want de punt mag niet als laatst
-// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
\ No newline at end of file
+// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
+
+// kreeg het niet voor elkaar met ternary operators omdat hierbij altijd valid anagepast wordt
+// terwijl je met volledige if-statements valid uitsluitend naar false kan zetten ipv 1 van de 2
+// function checkEmailValidity(email){
+//     let valid = true
+//     !email.includes("@") ? valid = false : valid = true
+//     console.log("contains @ = "+valid)
+//     email.charAt(email.length-1) === "." ? valid = false : valid = true
+//     console.log("last char is . ="+valid)
+//     email.includes(",") ? valid = true : valid = false
+//     console.log("contains , = "+valid)
+//
+//     return valid
+// }
+
+function checkEmailValidity(email){
+    let valid = true
+    if(!email.includes("@")){
+        valid = false
```

Comment:
In principe is het op dit punt al niet meer valide, dus kun je direct false teruggeven
```suggestion
        return false;
```